en/includes/guides/organization-management/manage-conflicts-in-organizations.md

```diff
@@ -0,0 +1,47 @@
+# Manage conflicts in organizations
+
+Now you can manage OAuth2 Applications and Roles inside the organizations. When managing these entities from the 
+organizations, organizations can have two types of entities.
+
+- Entities which are created when sharing
+- Entities which are managed directly in organizations
+
+## Entities which are created when sharing
+
+Root organizations can share applications with the organizations. This process will create the corresponding applications 
+and the associated roles in the organization.
+
+## Entities which are managed directly in organizations
+
+Organizations can have their own OAuth2 applications and roles that can be associated with those OAuth2 applications 
+managed in the organization.
+
+Since now both above entity types can be available in the organization level, there can be scenarios where these entities 
+might lead to some conflict.
```

Comment:
mention that resource conflict is possible due to name.
Sharing application renaming 
and sub org application existence cause for conflicts